builtin/credential/cert/path_certs.go

```diff
@@ -137,6 +165,10 @@ certificate.`,
 
 		HelpSynopsis:    pathCertHelpSyn,
 		HelpDescription: pathCertHelpDesc,
+		DisplayAttrs: &framework.DisplayAttributes{
+			Action:   "Create",
```

Comment:
this is what allows us to create a "nested item" -- i.e., a certificate nested under the cert auth method.
![image](https://user-images.githubusercontent.com/903288/88087740-3cff5a00-cb3e-11ea-8535-fdc06888883e.png)